src/components/VTable/VTable.vue

```diff
@@ -1,138 +1,112 @@
 <template>
-  <div>
-    <!-- <div className="lists-container">
```

Comment:
Id like to keep these comments here for now. Looking at making tables responsive as well, and looking at this for inspiration.